### PR TITLE
Run Compose instrumentation tests in presubmit

### PR DESCRIPTION
**Pull request by mihaipopa12**

### CODE REVIEW

File: constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayout.kt [hunk 1]
```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+
```
Comment (camaelon) [inline constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayout.kt:17]: probably unnecessary :)

File: .github/workflows/core.yml [hunk 2]
```diff
@@ -88,6 +88,10 @@ jobs:
           # Also don't use ${{ runner.os }} in the key so we don't re-assemble for UI tests.
           key: gradle-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/buildSrc/**') }}-${{ github.sha }}
 
+      - name: Install NDK
+        run: echo "y" | sudo /usr/local/lib/android/sdk/tools/bin/sdkmanager --install "ndk;20.0.5594570" --sdk_root=${ANDROID_SDK_ROOT}
```
Comment (camaelon) [inline .github/workflows/core.yml:92]: we need the NDK for this??

**Review by camaelon — APPROVED** [review]

File: constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayout.kt (hunk 1 above)
Comment (mihaipopa12) [reply constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayout.kt:17]: Yes, I just need a change to a *.kt file to make sure CI kicks off :)

File: .github/workflows/core.yml (hunk 2 above)
Comment (mihaipopa12) [reply .github/workflows/core.yml]: Nope, removed.

**Pull request merged**